Makefile

```diff
@@ -127,18 +129,29 @@ prepush: check-nginx
 .PHONY: release
 release: all deps docs $(SMF_MANIFESTS) check-nginx
 	@echo "Building $(RELEASE_TARBALL)"
+	@$(ROOT)/build/node/bin/node ./node_modules/.bin/kthxbai
 	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/mako
 	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/boot
 	@mkdir -p $(RELSTAGEDIR)/site
 	@touch $(RELSTAGEDIR)/site/.do-not-delete-me
 	cp -r $(ROOT)/bin \
 	    $(ROOT)/boot \
-	    $(ROOT)/build \
 	    $(ROOT)/build/nginx \
+	    $(ROOT)/lib \
 	    $(ROOT)/node_modules \
 	    $(ROOT)/sapi_manifests \
 	    $(ROOT)/smf \
+	    $(ROOT)/test \
 	    $(RELSTAGEDIR)/root/opt/smartdc/mako/
+	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/mako/build
+	cp -r $(ROOT)/build/scripts $(RELSTAGEDIR)/root/opt/smartdc/mako/build/
```

Comment:
Yuck. The handling of manta-scripts in the `make release` targets of Manta repos is baroque. Anyway, this is fine.

Reply:
Yeah, I'm not a fan of this stuff. If you have suggestions let me know.